modules/gdscript/gdscript_vm.cpp

```diff
@@ -397,32 +397,36 @@ void (*type_init_function_table[])(Variant *) = {
 #define OPCODES_OUT \
 	OPSOUT:
 #define OPCODE_SWITCH(m_test) goto *switch_table_ops[m_test];
+
 #ifdef DEBUG_ENABLED
 #define DISPATCH_OPCODE          \
 	last_opcode = _code_ptr[ip]; \
 	goto *switch_table_ops[last_opcode]
-#else
+#else // !DEBUG_ENABLED
 #define DISPATCH_OPCODE goto *switch_table_ops[_code_ptr[ip]]
-#endif
+#endif // DEBUG_ENABLED
+
 #define OPCODE_BREAK goto OPSEXIT
 #define OPCODE_OUT goto OPSOUT
-#else
+#else // !(defined(__GNUC__) || defined(__clang__))
 #define OPCODES_TABLE
 #define OPCODE(m_op) case m_op:
 #define OPCODE_WHILE(m_test) while (m_test)
 #define OPCODES_END
 #define OPCODES_OUT
 #define DISPATCH_OPCODE continue
+
 #ifdef _MSC_VER
 #define OPCODE_SWITCH(m_test)       \
 	__assume(m_test <= OPCODE_END); \
 	switch (m_test)
-#else
+#else //!_MSC_VER
 #define OPCODE_SWITCH(m_test) switch (m_test)
-#endif
+#endif // _MSC_VER
+
 #define OPCODE_BREAK break
 #define OPCODE_OUT break
-#endif
+#endif // defined(__GNUC__) || defined(__clang__)
```

Comment:
In my opinion, it makes sense to use empty lines for the outer `#if`:

```c++
#if defined(__GNUC__) || defined(__clang__)

...

#else // !(defined(__GNUC__) || defined(__clang__))

...

#endif // defined(__GNUC__) || defined(__clang__)
```

Reply:
Didn't understand. Can rephrase, please?

Reply:
I meant to separate the contents of the outer `#if` with additional empty lines, while leaving the inner `#if`s without empty lines. I.e. this:

```c++
#if defined(__GNUC__) || defined(__clang__)

...

#else // !(defined(__GNUC__) || defined(__clang__))

...

#endif // defined(__GNUC__) || defined(__clang__)
```

instead of this:

```c++
#if defined(__GNUC__) || defined(__clang__)
...
#else // !(defined(__GNUC__) || defined(__clang__))
...
#endif // defined(__GNUC__) || defined(__clang__)
```

Although to be honest it doesn't help much.

Reply:
IMO it really doesn't help